API/TranslatorReasonersAPI.yaml

```diff
@@ -72,6 +72,14 @@ paths:
       summary: Query reasoner via one of several inputs
       description: ''
       operationId: query
+      parameters:
+        - name: bypass_cache
+          in: query
+          description: Set to true in order to bypass any possible cached result message and try to answer the query over again
+          required: false
```

Comment:
`required` is false by default and could be removed here.

Reply:
fixed